src/containers/roadmap/index.js

```diff
@@ -4,30 +4,40 @@ import { states } from "./helpers.js";
 
 import {
     StyleRoadmap,
+    StyleRoadmapTitle,
     StyleRoadmapLine,
-    StyleRoadmapState
+    StyleRoadmapState,
+    StyleStateHeader,
+    StyleStateTitle,
+    StyleStateList,
+    StyleStateListItem
 } from './style';
 
-import roadmapLine from "assets/images/roadmap-line.svg";
-
 const statestLength = states.length;
 
 export default () => (
     <StyleRoadmap id="roadmap">
+        <StyleRoadmapTitle>ROADMAP</StyleRoadmapTitle>
```

Comment:
Text should be converted to uppercase with css `text-transform: uppercase`

Reply:
👍 
This is a bad habit, I used it when we used semantic ui... 😅